websites/P/Physics Wallah/presence.ts

```diff
@@ -13,8 +13,12 @@ presence.on("UpdateData", async () => {
 	const presenceData: PresenceData = {
 			largeImageKey: Assets.Logo,
 			startTimestamp: browsingTimestamp,
+			type: ActivityType.Watching,
 		},
-		{ pathname, href, search } = document.location;
+		{ pathname, href, search } = document.location,
+		[privacyMode] = await Promise.all([
+			presence.getSetting<boolean>("privacy"),
+		]);
```

Comment:
Since there is only one setting, it is cleaner to have the previous way without `Promise.all`.
```suggestion
		privacyMode = await presence.getSetting<boolean>("privacy");
```